lib/src/web_ui/code_input.dart

```diff
@@ -145,8 +202,75 @@ class CodeInput {
     return strIndex + 2;
   }
 
+  ///Find the list of words that contain the same prefix as currWord
+  List<String> _wordMatches(String currWord, bool fullWord) {
+    List<String> matchingWords = [];
+    int currLength = currWord.length;
+    for (String schemeWord in wordToDocs.keys) {
+      if (((schemeWord.length > currWord.length) && !fullWord) ||
+          schemeWord.length == currWord.length) {
+        if (schemeWord.substring(0, currLength) == currWord) {
+          matchingWords.add(schemeWord);
+        }
+      }
+    }
+    return matchingWords;
+  }
+
+  ///Finds and displays the possible words that the user may be typing
+  void _autocomplete() {
+    //Find the text to the left of where the typing cursor currently is
+    int cursorPos = findPosition(element, window.getSelection().getRangeAt(0));
+    List<String> inputText =
+        element.text.substring(0, cursorPos).split(RegExp("[(]+"));
+    List<String> matchingWords = [];
+    //Find the last word that was being typed
+    int currLength = 0;
+    List output = _currOp(element.text, cursorPos);
+    List output2 = _currOp(element.text, cursorPos, 2);
+    String findMatch = output[0];
+    bool fullWord = output[1];
```

Comment:
Likewise, Dart [prefers](https://www.dartlang.org/guides/language/effective-dart/design#prefer-a-non-imperative-verb-phrase-for-a-boolean-property-or-variable) that booleans are named with a form of "to be" most of the time.

`isFullWord` would probably be better here.